src/features/rewards/modalShowAdsHistory/index.tsx

```diff
@@ -0,0 +1,170 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  StyledAdsHeaderWrapper,
+  StyledAdsHistoryTitle,
+  StyledAdsInfoText,
+  StyledAdsInfoTextWrapper,
+  StyledAdsPerHourText,
+  StyledAdsSaveFiltered,
+  StyledNotSelectedLink,
+  StyledLink,
+  StyledSeparatorText,
+  StyledSubTitleText,
+  StyledText,
+  StyledThumbDownFilter,
+  StyledThumbDownNotSelectedFilter,
+  StyledWrapper
+} from './style'
+import Modal from '../../../components/popupModals/modal/index'
+import { getLocale } from '../../../helpers'
+import TableAdsHistory, { DetailRow } from '../tableAdsHistory/index'
+import { ThumbsupSIcon } from '../../../components/icons'
+
+export interface Props {
+  rows?: DetailRow[]
+  onClose?: () => void
+  id?: string
+  isMobile?: boolean
+  adsPerHour?: number
+  adsPerDay?: number
```

Comment:
let's improve naming on this one. This one is used to display `Ads you've received in the past 7 days`.  Maybe `totalDays`?

Reply:
adsPerDay is correct which matches bat-native-ads?

Reply:
this one is not the same

Reply:
![image](https://user-images.githubusercontent.com/9574457/62478420-5bdcd300-b7ab-11e9-8abe-eb6a4645c871.png)

this variable is for this title